mesh/nurbs.hpp

```diff
@@ -485,16 +497,43 @@ class NURBSPatchMap
 
 // Inline function implementations
 
-inline double &NURBSPatch::operator()(int i, int j)
+inline double &NURBSPatch::slice(int i, int j)
```

Comment:
Should there be index checks in the case `MFEM_DEBUG`?

Reply:
Indeed add a check on existence of data and the index limits.